src/flagpole/__init__.py

```diff
@@ -100,16 +101,18 @@ def dump_schema_to_file(self, file_path: str) -> None:
     @classmethod
     def from_feature_dictionary(cls, name: str, config_dict: dict[str, Any]) -> Feature:
         try:
+            # Disallow date_parsed from being set in the feature config
+            if "date_parsed" in config_dict:
+                config_dict.pop("date_parsed")
+
```

Comment:
Is there a better way to exclude this from our pydantic parsing? I tried to make this a private member variable with getter method, but pydantic seems to get a bit fussy about this.